### DISCUSSION TITLE
Couldn't connect to director

### DISCUSSION THREAD

**Discussion by mp006ltk** [composer/composer#12044] [General]
My EA-1 wasn't working, Tried rebooting and re-jailbreaking.  Even composer couldn't connect.  After that I used system manager and noticed a bunch of servers were status disabled  under the status tab next to network.  Once restarted the system worked again.  

My question is how did this happen?  Is it possible my original dealer got in and messed with it?

**Comment by xabbuh** [comment]
I guess you should rather use something like https://www.c4forums.com for your problem. It has nothing to do with the PHP dependency manager Composer.
